server/pkg/store/cassandra/db.go

```diff
@@ -0,0 +1,271 @@
+package cassandra
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/gocql/gocql"
+	"github.com/google/uuid"
+	cassandraclient "github.com/kube-tarian/kad/server/pkg/cassandra-client"
+	"github.com/kube-tarian/kad/server/pkg/types"
+)
+
```

Comment:
this code is refactored and move to app_store.go
update the changes in app_store.go